example/ios/PagerViewExample.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need that?

Reply:
This one is very Xcode-specific: it gets generated when I so much as _open_ the workspace in Xcode, and doesn't in any other scenario, i.e. when running via the CLI, so it's really up to you. To me, it makes sense maaaybe to include it, as some people will probably open the example in Xcode, but I don't have a strong opinion here, and also I'm not sure if there's a convention around files like that.

Reply:
https://stackoverflow.com/a/50368149

I think we can keep it, but also don't have strong opinion here

Reply:
@krozniata haven't seen that, makes sense, I'd say let's include it then